src/main/java/core/basesyntax/FigureBehaviour.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public interface FigureBehaviour {
+    void draw();
+
+    double getArea();
```

Comment:
checklist: Don’t put all behaviour into a single interface if the methods are conceptually different from each other.
All our classes and interfaces should have a single purpose - draw() and getArea() methods are not conceptually close to each other.

Reply:
will be remove in class Figure method getArea;